components/python/bleach/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 erwinlem
+#
+
+BUILD_BITS=NO_ARCH
+BUILD_STYLE=setup.py
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		bleach
+COMPONENT_VERSION=	5.0.1
+COMPONENT_SUMMARY=	An easy whitelist-based HTML-sanitizing tool
+COMPONENT_PROJECT_URL=	https://github.com/mozilla/bleach
+COMPONENT_FMRI=		library/python/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION=Development/Python
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL=	$(call pypi_url)
+COMPONENT_ARCHIVE_HASH=	sha256:0d03255c47eb9bd2f26aa9bb7f2107732e7e8fe195ca2f64709fcf3b0a4a085c
+COMPONENT_LICENSE=	Apache-2.0 
+COMPONENT_LICENSE_FILE=	LICENSE
+
+PYTHON_VERSIONS=	3.7 3.9
+
+include $(WS_MAKE_RULES)/common.mk
+
+## pytest fails, most likely outdated.
+#COMPONENT_TEST_CMD =	$(PYTHON) -m pytest
+#COMPONENT_TEST_DIR=	$(SOURCE_DIR)/tests
+#COMPONENT_TEST_ARGS =
+#COMPONENT_TEST_MASTER = $(COMPONENT_TEST_RESULTS_DIR)/results-all.master
+
```

Comment:
When dealing with python packages you'll need to analyse its meta data. Typically you'll find them in setup.py and/or setup.cfg.
For this package you'll find find
INSTALL_REQUIRES = [
    # html5lib requirements
    "six>=1.9.0",
    "webencodings",
]
in setup.py. Thus, you'll need to create packages for webencodings first as we don't have them yet. Furthermore, you'll need to add runtime dependencies in the manifest file for them an the appropriate python runtimes.
You can find some examples for this in other python packages, eg. in pytest.
This package also supports the tox test framework. You can find examples on how to use it also in other python packages, eg. wheel.